projects/components/src/quick-search/quick-search.component.spec.ts

```diff
@@ -392,6 +396,20 @@ describe('QuickSearchComponent', () => {
             expect(this.quickSearch.sectionTitles).toEqual(['section']);
         });
 
+        it('displays icon next to a section title', function (this: Test): void {
+            // Open
+            const simpleSearchProvider = new SimpleSearchProvider(false, 'user');
+            simpleSearchProvider.sectionName = 'new section';
+            this.quickSearchData.spotlightSearchService.registerProvider(simpleSearchProvider);
+            this.finder.hostComponent.spotlightOpen = true;
+            this.finder.detectChanges();
+            // Set search
+            this.quickSearch.searchInputValue = 'copy';
+            //
+            expect(this.quickSearch.searchResults.length).toBe(2);
```

Comment:
You can omit this so that the test and all expectations are really about displaying an icon